### PR TITLE
IMAGE  UPLODING FEATURES

### PR DESCRIPTION
**Pull request by atulgammastack**
Screenshot from 2022-09-07 12-54-18
In users  list users who have upload thier profile picture. you recognize by their profile picture.
Screenshot from 2022-09-07 12-56-01
user profile who have profile picture.
Screenshot from 2022-09-07 12-56-21
In post  uplaoded a mp4 video 
Screenshot from 2022-09-07 12-57-18
In post uploaded a image.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.0'
 
+gem 'active_storage_validations'
```
Comment (kalpana19sharma) [inline Gemfile]: Why do we need this gem?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -41,3 +41,13 @@ h1 {
...
+video{
+  width: 700px;
+}
+img{
+  width: 100px;
+}
+.img-circle {
+  width: 500px;
+  height: 300px;
+}
```
Comment (kalpana19sharma) [inline app/assets/stylesheets/application.scss]: Add new line

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]
@atulgammastack Please add screenshots of your changes in the PR description

**Comment by atulgammastack** [comment]
Screenshot from 2022-09-07 12-54-30
Screenshot from 2022-09-07 12-56-01
Screenshot from 2022-09-07 12-57-18
Screenshot from 2022-09-07 12-56-21
Screenshot from 2022-09-07 12-57-59

In user list users who how their profile pictures.

**Review by kalpana19sharma — APPROVED** [review]

**Pull request merged**